### DISCUSSION TITLE
넣으면 확실히 좋은 검증된 방법들

### DISCUSSION THREAD

**Discussion by sglee487** [bcaitech1/p4-mod-model_diet#38] [Announcements]
## 데이터 관련
* 이미지 사이즈 224 -> 64로 줄이면 MACs 10배가량 줄고

## 모델 관련
* pretrained weight를 불러오면 f1이 0.2~0.3 정도 올라간다.
* knowledge distillation 적용하면 f1이 0.08~ 0.1은 확실히 오른다.

## hyperparameter
* loss 함수는 CosineAnnealingWarmupRestarts
